server/serverConfigs.go

```diff
@@ -0,0 +1,129 @@
+package server
+
+import (
+	"fmt"
+	toml "github.com/pelletier/go-toml"
+	"github.com/spf13/viper"
+	"log"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+)
+
+var CONFIG_DIR_NAME = ".kelp"
+var cachedConfigDir string
+
+func getProjectFromCmd(cmd []string) string {
```

Comment:
a project is a folder full of config files.  Again, this might change but I had to make a decision to get it working for now.  This function helps the task locate it's project folder (stored in ~/.kelp for now).

Reply:
I think this is needlessly complex. we shouldn't need to introduce the concept of a project. you can assume that all the configs are in a single folder and the configs file names can be arbitrary.

i.e. a strategy file can be called `mirror.cfg` or `mirror_1.cfg` or `steve_config_file.cfg` etc.